adagrams/game.py

```diff
@@ -1,11 +1,157 @@
+import random
+
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
+score_dict = {
+    'A': 1, 
+    'B': 3, 
+    'C': 3, 
+    'D': 2, 
+    'E': 1, 
+    'F': 4, 
+    'G': 2, 
+    'H': 4, 
+    'I': 1, 
+    'J': 8, 
+    'K': 5, 
+    'L': 1, 
+    'M': 3, 
+    'N': 1, 
+    'O': 1, 
+    'P': 3, 
+    'Q': 10, 
+    'R': 1, 
+    'S': 1, 
+    'T': 1, 
+    'U': 1, 
+    'V': 4, 
+    'W': 4, 
+    'X': 8, 
+    'Y': 4, 
+    'Z': 10
+}
+
+
 def draw_letters():
-    pass
+# Create a list of letters using their frequencies
+# pull letters from the list using random.randint
+# create a new list from 10 random letters
+    list_letters = []   
+
+    for letter, freq in LETTER_POOL.items():
+        for i in range (0,freq):
+            list_letters.append(letter)
+
+    #return list_letters
+
+    hand = []
+    b = len(list_letters)
+    while len(hand)<10:
+        index  = random.randint(0,b-1)
+        new_letter = list_letters[index]
+        if hand.count(new_letter) < LETTER_POOL[new_letter]:
+            hand.append(new_letter)
+    return hand
+
+# Make all letter in word Capital
+# function returns True or False
 
 def uses_available_letters(word, letter_bank):
-    pass
 
-def score_word(word):
-    pass
+    #letter_bank = draw_letters()
+    copy_letter_bank = letter_bank[:]
+    word_up = word.upper()
+    
+    for letter in word_up:
+        if letter not in copy_letter_bank or word_up.count(letter) > copy_letter_bank.count(letter):
```

Comment:
Another great use of the count method. This particular compound conditional could get a little hard to read, so it could be separated into two different if statements if you wanted. You could also create a boolean to hold the result of `word_up.count(letter) > copy_letter_bank.count(letter)`, but only if you wanted to increase readability!